src/traits.rs

```diff
@@ -70,16 +70,21 @@ declare_primitive!(f64: (0.0)..1.0);
 /// An Enlargable::Larger value should be enough to calculate
 /// the sum (average) of a few hundred or thousand Enlargeable values.
 pub trait Enlargeable: Sized + Bounded + NumCast {
```

Comment:
Could you make this `pub(crate)` for clarity?

Reply:
I'm afraid, no. It's being used in a public interface even though it is not accessible, pretty much like a sealed trait.